arangod/Indexes/IndexFactory.cpp

```diff
@@ -76,6 +76,30 @@ struct InvalidIndexFactory : public IndexTypeFactory {
 }  // namespace
 
 namespace arangodb {
+namespace helpers {
+
+IndexId extractId(velocypack::Slice slice) noexcept {
+  auto id = IndexId::none();
+  auto value = slice.get(StaticStrings::IndexId);
```

Comment:
This is not `noexcept` if the Slice is not of type `Object`. Do we care here?

Reply:
It already checked above code, I can add assert